charts/unified-chart/test.values.yaml

```diff
@@ -36,7 +37,13 @@ prometheusRule:
 
 ingresses:
   tesla:
-    enabled: true
+    enabled: false
```

Comment:
in the test vlaues, we should enable the ingresses 
so we can always be sure we dont screw them up